packages/terra-form/src/Textarea.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+
+import './Textarea.scss';
+
+const propTypes = {
```

Comment:
I think the standard use case for these would be very rare, so I think they belong better on the customProps as opposed to being first class props. I'm not 100% locked into that idea though.

What do others think?